tests/seq_io/test_tag_select_intersect.h

```diff
@@ -35,17 +35,17 @@
 #ifndef TESTS_SEQ_IO_TEST_TAG_SELECT_INTERSECT_H_
 #define TESTS_SEQ_IO_TEST_TAG_SELECT_INTERSECT_H_
 
-#include <seqan/basic.h>
-#include <seqan/sequence.h>
-#include <seqan/seq_io.h>
-#include <seqan/bam_io.h>
-
 // ---------------------------------------------------------------------------
 // Assign tags to an output file based on the format of input file.
 // ---------------------------------------------------------------------------
 
 void testTransferTag()
 {
+    int offset = 1;
```

Comment:
Could you please add a comment to why and how offset is used.
It might not immediately become clear to new maintainers what it is meant for.